### PR TITLE
Fix e2e-node test panic

### LINKED ISSUE #57889: Received panic when execute e2e-node test
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
Received panic when I make a e2e-node test:

```bash
# make test-e2e-node FOCUS="should assign CPUs as expected based on the Pod spec" SKIP=""
   ... ...
•! Panic [8.361 seconds]
[sig-node] CPU Manager [Feature:CPUManager]
/root/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/framework.go:22
  With kubeconfig updated with static CPU Manager policy run the CPU Manager tests
  /root/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/cpu_manager_test.go:421
    should assign CPUs as expected based on the Pod spec [It]
    /root/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/cpu_manager_test.go:189

    Test Panicked
    assignment to entry in nil map
    /usr/local/go/src/runtime/panic.go:491

    Full Stack Trace
    	/usr/local/go/src/runtime/panic.go:491 +0x283
    k8s.io/kubernetes/test/e2e_node.enableCPUManagerInKubelet(0xc420f80000, 0x4)
    	/root/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/cpu_manager_test.go:148 +0x10e
    k8s.io/kubernetes/test/e2e_node.runCPUManagerTests.func1()
    	/root/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/cpu_manager_test.go:198 +0x14c
    k8s.io/kubernetes/vendor/github.com/onsi/ginkgo/internal/leafnodes.(*runner).runSync(0xc420c399e0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
    	/root/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/e2e_node_suite_test.go:135 +0x471
    testing.tRunner(0xc420a1ca50, 0x4558d60)
    	/usr/local/go/src/testing/testing.go:746 +0xd0
    created by testing.(*T).Run
    	/usr/local/go/src/testing/testing.go:789 +0x2de
    
------------------------------
```

**What you expected to happen**:
Do not panic.

**How to reproduce it (as minimally and precisely as possible)**:
Execute `e2e-node` test locally like:
```
# make test-e2e-node FOCUS="should assign CPUs as expected based on the Pod spec" SKIP=""
```

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by miaoyq**
Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

**What this PR does / why we need it**:
Execute e2e-node test locally like:
```bash
# make test-e2e-node FOCUS="should assign CPUs as expected based on the Pod spec" SKIP=""
```
received a panic see #57889
This pr fixes the panic.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #57889

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

File: test/e2e_node/cpu_manager_test.go [hunk 1]
```diff
@@ -145,6 +145,9 @@ func enableCPUManagerInKubelet(f *framework.Framework) (oldCfg *kubeletconfig.Ku
 	newCfg := oldCfg.DeepCopy()
 
 	// Enable CPU Manager using feature gate.
+	if newCfg.FeatureGates == nil {
```
Comment (spiffxp) [inline test/e2e_node/cpu_manager_test.go]: can you move this above the "Enable CPU Manager" comment? maybe right after the DeepCopy above?

Comment (miaoyq) [reply test/e2e_node/cpu_manager_test.go]: @spiffxp Thanks for review, Done. :)

**Pull request merged**